addons/sale/wizard/__init__.py

```diff
@@ -5,3 +5,4 @@
 from . import sale_make_invoice_advance
 from . import sale_order_cancel
 from . import sale_payment_link
+from . import sale_payment_register
```

Comment:
My bad, it should be first, after being renamed to `account_payment_register.py`